stories/example.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { expect, within } from "@storybook/test";
+import Home from "./example";
+
+const meta: Meta<typeof Home> = {
+	title: "Pages/Home",
+	component: Home,
+	parameters: {
+		layout: "fullscreen",
+	},
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Home>;
+
+export const Default: Story = {
+	play: async ({ canvasElement }) => {
+		const canvas = within(canvasElement);
+		await expect(canvas.findByText("app/page.tsx")).toBeInTheDocument();
```

Comment:
```suggestion
                const text = await canvas.findByText("app/page.tsx");
		await expect(text).toBeInTheDocument();
```

findByTextは `await` してあげないといけないかと思います！